inc/config.class.php

```diff
@@ -188,7 +188,7 @@ public function addDocumentTypes() {
 
       foreach ($extensions as $extension => $name) {
          $documentType = new DocumentType();
-         if (!$documentType->getFromDBByQuery("WHERE LOWER(`ext`)='$extension'")) {
+         if (!$documentType->getFromDBByCrit(['ext' => $extension])) {
```

Comment:
Doesn't the lower matters?

Reply:
well... no. Mysql is case insensitive when comparing strings !

Try to select an entry of glpi_documenttypes with a where on the column ext, and with an existing entry you type un upper case...